src/components/ItemCard/style.jsx

```diff
@@ -20,6 +20,45 @@ filter: drop-shadow(0px 0px 10px rgba(20, 23, 28, 0.1));
 transform: scale(1);
 transition: transform 200ms ease;
 
+& div.personal-details {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```

Comment:
when in "flex-direction: column" and not "row",
"align-items: center" will make the context horizontally centered.
if it was "row", "justify-content: center" would make it centered :S @OmerGery